superchic.sh

```diff
@@ -1,6 +1,7 @@
 package: SuperCHIC
 version: "%(tag_basename)s%(defaults_upper)s"
-tag: "v2.04"
+tag: "alice/v2.04"
+source: https://github.com/hcab14/SuperCHIC.git
```

Comment:
I would prefer (but not insist) to have ALICE code under the official ALICE account, i.e. alisw, and use others, s.a. alipwgmm, for development and/or staging. In the case of generators, changes should go via alipwgmm, however.